tests/scripts/all.sh

```diff
@@ -1029,7 +1035,7 @@ component_test_default_cmake_gcc_asan () {
 component_test_default_cmake_gcc_asan_new_bignum () {
     msg "build: cmake, gcc, ASan" # ~ 1 min 50s
     CC=gcc cmake -D CMAKE_BUILD_TYPE:String=Asan .
-    make CFLAGS="-D MBEDTLS_ECP_WITH_MPI_UINT"
+    make CFLAGS="$GLOBAL_CFLAGS -DMBEDTLS_ECP_WITH_MPI_UINT"
```

Comment:
```suggestion
    CC=gcc cmake -D CMAKE_BUILD_TYPE:String=Asan -DCMAKE_C_FLAGS="$GLOBAL_CFLAGS -DMBEDTLS_ECP_WITH_MPI_UINT" . 
    make 
```
Completely missed this. The way you have done this will override all standard C Flags set by CMakeFiles - extra flags should be passed in like so - this goes for all other cmake invocations.

Reply:
Sorry, tested this incorrectly. Either form is acceptable.